CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+
+## Creating a new sample
+
+### Pre-requisites
+
+**First**, look through existing samples to ensure you are not duplicating work. Also, look through Github issues if there is any similar work in progress (ie. issues with PRs associated to them).
+
+**Second**, once you've confirmed this, you need to decide whether you are creating either an End-to-End Sample or a Single Technology Sample.
+
+1. [End-to-End Samples](/e2e_samples/) - Reference-architecture style DataOps samples typically utilizing a number of Microsoft services and technologies.
+   - **Purpose**: Showcase a number of DataOps key concepts in a concrete, end-to-end data pipeline.
+
+2. [Single Technology Samples](/single_tech_samples/) - Smaller, modular samples focusing on a single Azure service showcasing one or two DataOps key concepts.
+   - **Purpose**: allows Developers and Data Engineers to easily choose and reuse individual samples depending on their requirements without the complexity of an end-to-end sample. Being small and modular also allows for showcasing different implementations for the same concept.
+
+3. [Utilities](/utilities/) - Reusable, standalone scripts that help with a specific operational task.
+   - **Purpose**: Easily find helpful scripts used in common tasks to manage a data engineering system.
+
+**Third**, decide which [key concepts](https://docs.microsoft.com/en-us/azure/architecture/framework/) are you showcasing in your sample. See [Microsoft Azure Well-Architected Framework](https://docs.microsoft.com/en-us/azure/architecture/framework/) for additional information. If you've decided to build a Single Technology Sample, decide which technology will be showcasing the key concepts.
+
+**Fourth**, locate where in the repository structure your sample will sit. Decide on the name of the sample -- short descriptive names are best. It is recommended to **not** choose a name too lengthy and complicated as it will be used across the repo (folder name, script paths, deep-linking in docs, labels, etc.).
+
+Repo Structure:
+
+```text
+/e2e_samples
+  /<name-of-e2e-sample-1>
+/single_tech_samples
+  /azuresql
+  /cosmosdb
+  /fabric
+  /databricks
+    /<name-of-singletech-sample-1>
+    /<name-of-singletech-sample-2>
+  ...
+/utilities
+  /fabric
+    /<name-of-utility-1>
+    /<name-of-utility-2>
+  /databricks
+```
+
+**Fifth**, Create a Github issue using the "New Sample Setup" template and fill in the information required. This issue template also contains a detailed checklist to guide you in your sample development.
+
+### Getting Started
+
+Once you have done all of the above, see this [sample structure](/docs/sample_project_structure/README.md) to get started. Use that structure as a starting point.
+
+#### Your sample's README
+
+This is arguably the *most important* part of your sample. In general, it is recommended to put most of the content in a single README to avoid content being "lost" and skipped over through deep linking.
```

Comment:
```suggestion
This is arguably the *most important* part of your sample. It is recommended to consolidate most content in a single README to prevent information from being 'lost' or overlooked through through deep linking.
```